valispace/__init__.py

```diff
@@ -162,7 +182,14 @@ def get_vali_list(self, workspace_id=None, workspace_name=None, project_id=None,
 		if vali_marked_as_impacted:
 			url = self.__increment_url(url) + "valis_marked_as_impacted={}".format(vali_marked_as_impacted)
 		response = requests.get(url, headers=self.get_request_headers())
-		return response.json()
+
+		if response.status_code != 200:
```

Comment:
Are these prints for debug or should it stay in the code?

Reply:
These prints can be left here, I think.